nix/modules/fedimintd.nix

```diff
@@ -236,6 +251,23 @@ in
     in
     mkIf (eachFedimintd != { }) {
 
+      assertions = [
```

Comment:
Do you happen to know what is evaluating assertions to actually trigger them to fail? Is this something module system supports internally?